<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 70, 39);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>751</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="ClockInWidget" name="clockInPage">
    <attribute name="title">
     <string>刷卡</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/pic/marry.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="ManageWidget" name="manPage">
    <attribute name="title">
     <string>管理</string>
    </attribute>
   </widget>
   <widget class="ShopWidget" name="shopPage">
    <attribute name="title">
     <string>消费</string>
    </attribute>
   </widget>
   <widget class="ReadWidget" name="readPage">
    <attribute name="title">
     <string>阅览计费</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/pic/marry.png);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>741</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/title.png);</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>268</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DB Server:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reconnectBtn">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Reconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/flower.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/flower.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/flower.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/flower.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClockInWidget</class>
   <extends>QWidget</extends>
   <header>ClockInWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ManageWidget</class>
   <extends>QWidget</extends>
   <header>ManageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShopWidget</class>
   <extends>QWidget</extends>
   <header>ShopWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReadWidget</class>
   <extends>QWidget</extends>
   <header>readwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
